sections/image-banner.liquid

```diff
@@ -79,6 +84,11 @@
         else
           assign sizes = '100vw'
         endif
+        if section.settings.image_behavior == 'ambient'
+          assign widths = '450, 660, 900, 1320, 1800, 2136, 2400, 3600, 7680'
```

Comment:
The `sizes` property is probably the one you want to change here instead (or as well) as the `widths` one.  It's the property that determines which width gets picked so should be scaled by 1.2 as well.

Suggestion:
```
{%- liquid
  if section.settings.image_behavior == 'ambient'
    assign half_width = '60vw'
    assign full_width = '120vw'
    assign two_column_sizes = '(min-width: 750px) 60vw, 120vw'
    assign widths = '450, 660, 900, 1320, 1800, 2136, 2400, 3600, 7680'
  else
    assign half_width = '50vw'
    assign full_width = '100vw'
    assign two_column_sizes = '(min-width: 750px) 50vw, 100vw'
    assign widths = '375, 550, 750, 1100, 1500, 1780, 2000, 3000, 3840'
  endif
-%}
```
Then for each image:
```
        if section.settings.image != blank and section.settings.stack_images_on_mobile
          assign sizes = two_column_sizes
        elsif section.settings.image_2 != blank
          assign sizes = half_width
        else
          assign sizes = full_width
        endif
```

Reply:
Thanks, that makes sense! I pushed this change in 01dc560.